src/main/views/home.njk

```diff
@@ -1,10 +1,5 @@
-<html lang="en">
```

Comment:
Do we need home.njk anymore? it could be task-list, coouldn't it?

Reply:
Yes, we also don't need the `home.ts`route also. But they were both here when I branched of master so decided to keep.